### ISSUE TITLE
[Test][cypress] Edit Profile Details

### ISSUE THREAD

**Issue by elraphty** [stakwork/sphinx-tribes-frontend#401]
### Context

Currently, we have a cypress command to [create a user profile](https://github.com/stakwork/sphinx-tribes-frontend/blob/master/cypress/support/commands.ts#L43C1-L99C1) but we want to assert that we can edit the user's profile details. 

So what we want to do is use [Login User](https://github.com/stakwork/sphinx-tribes-frontend/blob/master/cypress/support/commands.ts#L43C1-L99C1) to create a user profile, click on the user icon, and edit the user profile, change the user description, Twitter username, Github username and Coding languages.

### File to create
`sphinx-tribes-frontend/cypress/e2e/14_edit_profile_details.cy.ts`
there will be modifications to `sphinx-tribes-frontend/cypress/support/commands.ts`

### Acceptance Criteria

- Given a user is signed in
- Click on the User profile icon/username, to lead you to the user profile
- Click on the Edit button
- Edit the user details 
- Save the updated details 
- Then Assert that the updated details become the new user details.

The edit profile modal is shown below

![image](https://i.ibb.co/02V1Y8n/Screenshot-2024-03-14-at-13-51-23.png)

- [ ] I have rebased and tested locally before submitting my PR
- [ ] I can submit a PR within 2 days of taking the bounty

### References
Example cypress test
`https://github.com/stakwork/sphinx-tribes-frontend/blob/master/cypress/e2e/0_create_bounty.cy.ts`

**Comment by saithsab877** [comment]
assign

**Comment by ColdMice** [comment]
@elraphty can you please assign this to me. Thanks

**Comment by MahtabBukhari** [comment]
assign me?

**Comment by AbdulWahab3181** [comment]
Assign me

**Comment by elraphty** [comment]
> Hi @elraphty could you please assign me to the task?

@saithsab877 written any cypress tests yet?

**Comment by saithsab877** [comment]
> > Hi @elraphty could you please assign me to the task?
> 
> @saithsab877 written any cypress tests yet?

Yes, https://github.com/stakwork/sphinx-tribes-frontend/pull/400

**Comment by elraphty** [comment]
> #400

Oh! sorry you are Abubakar on Sphinx Chat, I'm assigning.

**Comment by saithsab877** [comment]
> > #400
> 
> Oh! sorry you are Abubakar on Sphinx Chat, I'm assigning.

Thanks

**Comment by saithsab877** [comment]
Hi @elraphty,

After successfully editing the user profile in the Cypress test, as I click on the save button, this scanner popup appears. I am puzzled as to what this is because it is not present on the actual website. Could you please guide me on how to tackle this issue?

![image](https://github.com/stakwork/sphinx-tribes-frontend/assets/156602406/7afb99c8-63a5-484f-b6e1-a716c236581c)

**Comment by saithsab877** [comment]
Hi @elraphty,
Please guide me?

**Comment by elraphty** [comment]
> Hi @elraphty,
> 
> After successfully editing the user profile in the Cypress test, as I click on the save button, this scanner popup appears. I am puzzled as to what this is because it is not present on the actual website. Could you please guide me on how to tackle this issue?
> 
> ![image](https://private-user-images.githubusercontent.com/156602406/313173458-7afb99c8-63a5-484f-b6e1-a716c236581c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NzI5OTAsIm5iZiI6MTcxMDc3MjY5MCwicGF0aCI6Ii8xNTY2MDI0MDYvMzEzMTczNDU4LTdhZmI5OWM4LTYzYTUtNDg0Zi1iNmUxLWE3MTZjMjM2NTgxYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOFQxNDM4MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYWJlZDNiZmY3NzNiNjE1ZWFkODBhYTUzZjU4NjdkMWUwZDYzNTZlNWI1ZTU1MTUzNjc2ZTEyOGI3YWNiMzg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cQqnLYQmXQRUWHG2OxmVXsWcEQ192WECUF-zhd0k_kI)

@saithsab877 make sure you are not calling login during the process

**Comment by saithsab877** [comment]
> > Hi @elraphty,
> > After successfully editing the user profile in the Cypress test, as I click on the save button, this scanner popup appears. I am puzzled as to what this is because it is not present on the actual website. Could you please guide me on how to tackle this issue?
> > ![image](https://private-user-images.githubusercontent.com/156602406/313173458-7afb99c8-63a5-484f-b6e1-a716c236581c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NzI5OTAsIm5iZiI6MTcxMDc3MjY5MCwicGF0aCI6Ii8xNTY2MDI0MDYvMzEzMTczNDU4LTdhZmI5OWM4LTYzYTUtNDg0Zi1iNmUxLWE3MTZjMjM2NTgxYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOFQxNDM4MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYWJlZDNiZmY3NzNiNjE1ZWFkODBhYTUzZjU4NjdkMWUwZDYzNTZlNWI1ZTU1MTUzNjc2ZTEyOGI3YWNiMzg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cQqnLYQmXQRUWHG2OxmVXsWcEQ192WECUF-zhd0k_kI)
> 
> @saithsab877 make sure you are not calling login during the process

Please check my code. I not calling login :

![KAINb5l](https://github.com/stakwork/sphinx-tribes-frontend/assets/156602406/35b6ff04-f30e-4a6c-9592-95e2eb2aee9b)

**Comment by saithsab877** [comment]
I create a PR then you check it.

**Comment by cybercraftsolutionsllc** [comment]
Submitted PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1612

Added Cypress coverage for editing profile details:
- logs in and opens Edit Profile
- updates description, Twitter, Github, and Coding Languages
- saves and asserts the saved toast plus updated description

Validation:
- `npx tsc --noEmit --target es2015 --types cypress,node --moduleResolution node cypress\global.d.ts cypress\e2e\14_edit_profile_details.cy.ts`
- `npx eslint cypress\e2e\14_edit_profile_details.cy.ts --max-warnings 100`
- `git diff --check`
